3.BinarySearch/search.js

```diff
@@ -0,0 +1,20 @@
+function binarySearch(arr, target) {
+  var half = 0;
+  var left = 0;
+  var right = arr.length - 1;
+  while(true) {
```

Comment:
I'm not a fan of `while (true)`. inside of loop you have condition to finish loop, please use it instead of true